config/packages.yml

```diff
@@ -86,19 +71,8 @@ drupal/acquia_search:
 
 acquia/blt:
   type: composer-plugin
-  core_matrix:
-    '>=9.0.0':
-      version: 12.x
-      version_dev: 12.x-dev
-    '>=8.8.0':
-      version: 11.x
-      version_dev: 11.x-dev
-    8.7.x:
-      version: 10.x
-      version_dev: 10.x-dev
-    8.6.x:
-      version: 9.x
-      version_dev: 9.x-dev
+  version: 13.x
+  version_dev: 13.x-dev
```

Comment:
I'm not sure if this will work because of a stale BLT branch alias that I can't get rid of without breaking BC. Need to look into it.

Reply:
I created a 12.x branch and updated the dev-main alias to be 13.x-dev, which I think should fix it...